python/artm/dictionary.py

```diff
@@ -106,7 +107,9 @@ def load_text(self, dictionary_path, encoding='utf-8'):
         self._reset()
         dictionary_data = messages.DictionaryData()
         with codecs.open(dictionary_path, 'r', encoding) as fin:
-            dictionary_data.name = fin.readline().split(' ')[1][0: -1]
+            first_str = fin.readline()[: -1].split(' ')
```

Comment:
Here we made old text dictionaries incompatible, but I don't think that it's very bad:

1) people mostly store dictionaries in binary format, text is smth temporary (binary format exporting didn't change, only import was fixed)
2) text dictionary can be simply updated by hand (by appending " num_items: <num_items>" to the first line)